setup.py

```diff
@@ -46,5 +54,12 @@
         'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7',
         'Programming Language :: Python :: 3 :: Only',
-    ]
+    ],
+    project_urls={
+        'Bug Reports': 'https://github.com/djangonauts/django-rest-framework-gis/issues',
+        'Continuous Integration': 'https://travis-ci.org/djangonauts/django-rest-framework-gis',
+        'Mailing List': 'https://groups.google.com/forum/#!forum/django-rest-framework-gis',
+        'Code Coverage': 'https://coveralls.io/github/djangonauts/django-rest-framework-gis',
+        'Source Code': 'https://github.com/pypa/sampleproject/',
```

Comment:
this line doesn't look right and it's also causing the CI build to fail for some reason I don't understand

Reply:
If you are talking about 'Code Coverage' line I've fixed the line with the [second commit ](https://github.com/djangonauts/django-rest-framework-gis/pull/182/commits/f170e12cb541b915b69b544df253ed639da7bedd) and the [second build](https://travis-ci.org/djangonauts/django-rest-framework-gis/builds/455914808) passed.

Reply:
@pauloxnet `https://github.com/pypa/sampleproject/` is wrong, isn't it?
Should be: https://github.com/djangonauts/django-rest-framework-gis

Reply:
You're right. I've fixed it!